src/firefly/js/visualize/ui/StretchDropDownView.jsx

```diff
@@ -90,10 +90,12 @@ function stretchByType(pv,currRV,sType,min,max) {
 }
 
 
-
+/*
+ *  LZ 9/21/16  DM-7695
+ */
 export function StretchDropDownView({plotView:pv}) {
     var enabled= pv ? true : false;
-    var rv= primePlot(pv).plotState.getRangeValues();
+    var rv= primePlot(pv).plotState.getPrimaryRangeValues();//getRangeValues();//LZ 9/21/16
```

Comment:
This change should not be any different than what was there before. In the JavaScript `PlotState`, `getPrimaryRangeValues()` is the same as `getRangeValues()` when passed with no parameters. Making this change should make no difference.

Also, I plan to deprecate `getPrimaryRangeValues()` on the JS side.